test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -691,9 +691,11 @@ class ReliefDetailsControllerSpec
       behave like noPrivateResidentsReliefBehaviour(() => performAction(Seq.empty))
 
       "show a form error" when {
+        type Message    = String
+        type MessageKey = String
 
         def test(data: (String, String)*)(
-          expectedErrorMessageKey: String
+          expectedErrorMessageOrKey: Either[MessageKey, Message]
```

Comment:
you could just take the key as one parameter and the arguments as another parameter and pass them both to `messageFromMessageKey` - that way you can avoid an `Either`

Reply:
Done.